src/main/java/seedu/parser/ParseAdd.java

```diff
@@ -90,13 +98,14 @@ public String fetchDescription() throws EmptyStringException, StringIndexOutOfBo
 
     public double fetchDouble() throws InvalidCharacterInAmount, EmptyStringException,
```

Comment:
We need to add this method into ParseLendBorrow as well. Not to mention txtfile. I can implement your idea on the aforementioned.